3-enrich/emr-etl-runner/lib/snowplow-emr-etl-runner/emr_job.rb

```diff
@@ -157,6 +157,14 @@ def initialize(debug, staging, enrich, staging_stream_enrich, shred, es, archive
         csbe = config[:aws][:s3][:buckets][:enriched]
         csbs = config[:aws][:s3][:buckets][:shredded]
 
+        # Clear HDFS if persistent jobflow has been found
+        if found_persistent_jobflow
+          submit_jobflow_step(get_rmr_step(ENRICH_STEP_INPUT, standard_assets_bucket, "Empty Raw HDFS"), use_persistent_jobflow)
+          submit_jobflow_step(get_rmr_step(ENRICH_STEP_OUTPUT, standard_assets_bucket, "Empty Enriched HDFS"), use_persistent_jobflow)
+          submit_jobflow_step(get_rmr_step(SHRED_STEP_OUTPUT, standard_assets_bucket, "Empty Shredded HDFS"), use_persistent_jobflow)
```

Comment:
could we combine those as one step?

Reply:
So the underlying script would accept N HDFS path arguments and loop through them?  Yeah that can be done easily enough.

Reply:
or just rm `/local/snowplow`